src/gui/torrentcontentfiltermodel.cpp

```diff
@@ -90,7 +90,7 @@ bool TorrentContentFilterModel::lessThan(const QModelIndex &left, const QModelIn
     TorrentContentModelItem::ItemType rightType = m_model->itemType(m_model->index(right.row(), 0, right.parent()));
 
     if (leftType == rightType)
-      return Utils::String::naturalCompareCaseSensitive(vL, vR);
+      return Utils::String::naturalCompareCaseInsensitive(vL, vR);
```

Comment:
> We are used to case-sensitive collation.

how so?
maybe this is an exception, but my nemo file manager does sort files/folders case-insensitively.